packages/atomic-layout/src/hooks/useResponsiveComponent.spec.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * @jest-environment node
+ */
+import React from 'react'
+import { renderToString } from 'react-dom/server'
+import useResponsiveComponent from './useResponsiveComponent'
+
+const Component = useResponsiveComponent((props) => {
+  return <img {...props} />
+})
+
+describe('useResponsiveComponent', () => {
+  describe('given rendered on a server', () => {
+    let html: ReturnType<typeof renderToString>
+
+    beforeAll(() => {
+      html = renderToString(
+        <Component src="image.png" altMd="Image" titleLgDown="Title" />,
+      )
+    })
+
+    it('should have responsive prop with default breakpoint', () => {
+      expect(html).toContain('src="image.png"')
+    })
+
+    it.skip('should have responsive prop with "down" behavior', () => {
```

Comment:
will it be good to add a comment for the skipped test?

Reply:
Will create the issue and add the comment before the test 👍

Reply:
Opened the respective issue: https://github.com/kettanaito/atomic-layout/issues/284

Reply:
https://github.com/kettanaito/atomic-layout/issues/284